Makefile

```diff
@@ -16,41 +16,41 @@ PLATFORM_ID = $$( uname -s )
 endif
 PLATFORM = $$( \
 	case $(PLATFORM_ID) in \
-		( Linux | FreeBSD | OpenBSD | NetBSD | Windows) echo $(PLATFORM_ID) ;; \
+		( Linux | FreeBSD | OpenBSD | NetBSD | Windows | Darwin ) echo $(PLATFORM_ID) ;; \
 		( * ) echo Unrecognized ;; \
 	esac)
 
 CTAGS = $$( \
 	case $(PLATFORM_ID) in \
-		( Linux ) echo "ctags" ;; \
+		( Linux | Darwin ) echo "ctags" ;; \
 		( FreeBSD | OpenBSD | NetBSD | Windows) echo "exctags" ;; \
```

Comment:
Are ``ctags`` available are they called ``exctags`` on MacOS like they are on the BSDs?

Reply:
It's correct as it is. `brew install ctags` installs Exuberant Ctags as `ctags`